tensorboard/webapp/runs/types.ts

```diff
@@ -66,3 +66,12 @@ export interface RegexGroupBy {
 }
 
 export type GroupBy = BaseGroupBy | RegexGroupBy;
+
+/**
+ * The runs-related state created by deserializing a URL.
+ */
+export interface URLDeserializedState {
+  runs: {
+    groupBy: GroupBy | null;
```

Comment:
Is this needed to be changed once we separate gorupBy with "groupByKey" and "regexString"?

Reply:
No, it does not need to change; public interface outside of the redux still can remain `GroupBy` without destructing it.